client/socketAddr.js

```diff
@@ -0,0 +1 @@
+window.socketAddr = 'http://localhost:8081';
```

Comment:
why are we referencing the socket directly? should it not be a reference from your portfile? 

to: @admoores @nihilates 
ref: @faiz121